src/components/FeatureViews/Registration/RegistrationCourseEnrollments.js

```diff
@@ -32,261 +41,349 @@ import gql from "graphql-tag";
-                  firstName
-                  lastName
-                  email
-                  id
-                }
-                phoneNumber
-              }
-              user {
-                firstName
-                lastName
-                email
-                id
-              }
-              school {
-                name
-              }
-            }
+  query EnrollmentDetails($courseId: ID!) {
+    enrollments(courseId: $courseId) {
+      student {
+        primaryParent {
+          user {
+            firstName
+            lastName
+            email
             id
-         }
-	}
-	`;
+          }
+          phoneNumber
+        }
+        user {
+          firstName
+          lastName
+          email
+          id
+        }
+        studentschoolinfoSet {
+          textbook
+          teacher
+          name
+        }
+      }
+      id
+    }
+  }
+`;
 
 const TableToolbar = (
-    <TableHead>
-        <TableRow>
-            {["Student", "Parent", "Phone", "Upcoming Status", ""].map((heading) => (
-                <TableCell align="left" key={heading} padding="default">
-                    {heading}
-                </TableCell>
-            ))}
-        </TableRow>
-    </TableHead>
+  <TableHead>
+    <TableRow>
+      {["Student", "Parent", "Phone", "", "", "", "", ""].map((heading) => (
+        <TableCell align="left" color="color" key={heading} padding="default">
+          {heading}
+        </TableCell>
+      ))}
+    </TableRow>
+  </TableHead>
 );
 
-const RegistrationCourseEnrollments = ({ courseID, maxCapacity, courseTitle }) => {
-    const dispatch = useDispatch();
+const RegistrationCourseEnrollments = ({
+  courseID,
+  maxCapacity,
+  courseTitle,
+}) => {
+  const dispatch = useDispatch();
 
-    const [expanded, setExpanded] = useState({});
+  const [expanded, setExpanded] = useState({});
 
-    const [studentMenuAnchorEl, setStudentMenuAnchorEl] = useState(null);
-    const [unenroll, setUnenroll] = useState({
-        "enrollment": null,
-        "open": false,
-    });
+  const [studentMenuAnchorEl, setStudentMenuAnchorEl] = useState(null);
+  const [unenroll, setUnenroll] = useState({
+    enrollment: null,
+    open: false,
+  });
 
-    const { data, loading, error } = useQuery(GET_ENROLLMENT_DETAILS, {
-        variables: { courseId: courseID }
-    });
-    
+  const { data, loading, error } = useQuery(GET_ENROLLMENT_DETAILS, {
+    variables: { courseId: courseID },
+  });
 
-    // TODO: need to update when Session queries are live
-    // const sessionStatus = useSessions("month", 0)
-    // const currentMonthSessions = sessionArray(sessions);
-    // const upcomingSess = upcomingSession(currentMonthSessions || [], courseID);
+  // TODO: need to update when Session queries are live
+  // const sessionStatus = useSessions("month", 0)
+  // const currentMonthSessions = sessionArray(sessions);
+  // const upcomingSess = upcomingSession(currentMonthSessions || [], courseID);
```

Comment:
clean up unused comments. Also, you should be able to uncomment these because session query is live?

Reply:
Decided to keep for further use.